BaragonAgentService/src/main/java/com/hubspot/baragon/agent/managed/LifecycleHelper.java

```diff
@@ -246,6 +246,64 @@ public void applyCurrentConfigs() throws AgentServiceNotifyException {
     }
   }
 
+  public void applyAll(List<Optional<Pair<ServiceContext, Collection<BaragonConfigFile>>>> toApply) {
+    toApply.parallelStream().forEach(item -> {
+      try {
+        configHelper.bootstrapApply(item.get().getKey(), item.get().getValue());
+      } catch (Exception e) {
+        LOG.error(String.format("Caught exception while applying %s during bootstrap", item.get().getKey().getService().getServiceId()), e);
```

Comment:
quick note that LOG.(info|warn|etc) uses `{}` as the placeholder for arguments, unlike String.format which uses `%`

Reply:
e.g. This can more simply be written as 
```java
        LOG.error("Caught exception while applying {} during bootstrap", item.get().getKey().getService().getServiceId(), e);
```